pkg/jtracer/jtracer.go

```diff
@@ -88,6 +88,8 @@ func initOTEL(ctx context.Context, svc string) (*sdktrace.TracerProvider, error)
 			))
 	})
 
+	otel.SetTracerProvider(tracerProvider)
```

Comment:
Moved here so we just set the global tracer in one place, and removes the otel dependencies from the respective files that use them.

I didn't see, or could think of, a use case where we _wouldn't_ want to set the global tracer when instantiating a jtracer. 

Let me know if that's not the case.